examples/trl/reward_modeling.py

```diff
@@ -181,6 +181,7 @@ class ScriptArguments:
 tokenizer.pad_token = tokenizer.eos_token
 model.config.pad_token_id = tokenizer.eos_token_id
 model.config.use_cache = not script_args.gradient_checkpointing
+model.config.use_fused_rope = False
```

Comment:
No need for `model.config.use_fused_rms_norm = False` here?

Reply:
the condition checked is:
if hasattr(self.config, "use_fused_rms_norm") and self.config.use_fused_rms_norm is False:

only 1 place adds use_fused_rms_norm=False (ppo.py). For rest of them hasattr(self.config, "use_fused_rms_norm")  will fail and use_fused_rms_norm wont be set to False

This is really just a way to disable fused_rms in ppo